src/main/java/nextstep/courses/domain/ImageType.java

```diff
@@ -0,0 +1,25 @@
+package nextstep.courses.domain;
+
+public enum ImageType {
+    GIF,
+    JPG,
+    JPEG,
+    PNG,
+    SVG;
+
+    public static ImageType typeCheck(String imageType) {
```

Comment:
해당 메서드는 정적 팩토리 메서드라고 볼 수 있으니, 메서드 이름을 `from()` 정도로 지어보면 어떨까요?
('이펙티브 자바'에서는 보통 매개변수가 하나인 경우 `from()`을 사용하도록 가이드하고 있습니다)
```suggestion
    public static ImageType from(String imageType) {
```

Reply:
지식의 출처 기반이 맨 땅에 헤딩이나 구글링이었어서 네이밍 컨벤션에도 약하긴 합니다ㅋㅋㅋ 이번 과정 끝나면 이전에는 와닿지 않아서 미루었던 책들을 읽어보면서 정리해봐야겠네요